index.Rmd

```diff
@@ -1,105 +1,299 @@
+
+# Data and Results
+## PART1: Selection of thhe landing/mining sites
 
-Narrative: Clear narrative description of the data sources and methods. Includes data from at least two sources that were integrated / merged in R.
+The goal here is to detect the site where we could have more water/ water ice/ volatiles materials. I applied some corrections to satelitte images I downloaded. For question of time I will only show what I did in only one site here but all sites I selected to be future landing/mining site are shown later in the second part of this project.
 
-Code: The code associated with the project is well organized and easy to follow. Demonstrates mastery of R graphics and functions.
+I loaded required packages
+
+```{r, message=F, warning=F}
+library(raster)
+library(tidyverse)
+library(sf)
+library(rpart)
+library(rpart.plot)
+library(rasterVis)
+library(mapedit)
+library(mapview)
+library(caret)
+library(forcats)
+library(sp)
+library(raster)
+```
 
-Data: The underlying data are publicly accessible via the web and downloaded/accessed within the Rmd script. If you want to use your own data, you must make it available on a website (e.g. Figshare) so that others are able to re-run your code.
+I prepare the data by reading in the different bands that comprise the satelitte imagery. Each band refers to different spectrum.
 
-You can do bullets like this:
 
-* The first most important thing
-* The second most important thing
-* The third most important thing
+```{r, message=F, warning=F}
+Band1 <- ("B02.tif")
+Band2 <- ("B03.tif")
+Band3 <- ("B04.tif")
+Band4 <- ("B05.tif")
+Band5 <- ("B06.tif")
+Band6 <- ("B07.tif")
+Band7 <- ("B08.tif")
+
+band1 <- ("b01.tiff")
+band2 <- ("b02.tiff")
+band3 <- ("b03.tiff")
+band4 <- ("b04.tiff")
+band5 <- ("b05.tiff")
+band6 <- ("b06.tiff")
+band7 <- ("b07.tiff")
+band8 <- ("b08.tiff")
+band9 <- ("b09.tiff")
+```
 
-You can do numbers like this:
 
-1. The first most important thing
-2. The second most important thing
-3. The third most important thing
 
-See [http://rmarkdown.rstudio.com/](http://rmarkdown.rstudio.com/) for all the amazing things you can do.
+```{r, message=F, warning=F}
+library(raster)
+#This first part of the plot is an example on how it works with Landsat data for Earth
+Band1 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/B02.tif")
```

Comment:
you will need to make these available in your repository (or download from a public URL)